### PR TITLE
CI: Add external link validation

### PR DESCRIPTION
**Pull request by evaldofelipe**
Add [awesome_bot](https://github.com/dkhamsing/awesome_bot/), tool for check the links integrity.
Signed-off-by: uma-mining <contato@evaldofelipe.com>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,16 +2,30 @@ version: 2.1
 jobs:
   test:
     docker:
-      - image: circleci/node:lts
+      - image: dkhamsing/awesome_bot
+    parallelism: 50
     steps:
+      - checkout
       - run:
-          name: Test dependencies
```
Comment (chrismaree) [inline .circleci/config.yml:8]: do we not need the `Test dependencies` job that was here before?

File: .circleci/config.yml [hunk 2]
```diff
@@ -2,16 +2,30 @@ version: 2.1
...
+          name: Test links
           command: |
-            echo "No tests"
-            circleci-agent step halt;
+            cd docs/
+            find . -name '*.md' | circleci tests split > /tmp/test-docs
```
Comment (chrismaree) [inline .circleci/config.yml:13]: this is really cool.

why do you want to allow 403?

**Review by chrismaree — COMMENTED** [review]
Awesome stuff @evaldofelipe !

File: .circleci/config.yml (hunk 1 above)
Comment (evaldofelipe) [reply .circleci/config.yml:8]: It's not necessary, the past job was used as a empty job until we have this pipeline ready.

File: .circleci/config.yml (hunk 2 above)
Comment (evaldofelipe) [reply .circleci/config.yml:13]: a bunch of links from etherscan replies with 403 but after that they redirect and are working fine, so it's necessary add that role.

Comment (mrice32) [reply .circleci/config.yml:13]: Is it possible to allow 403 only for the etherscan domain?

**Review by mrice32 — COMMENTED** [review]
Looks great! One question on @chrismaree's comment.

File: .circleci/config.yml (hunk 2 above)
Comment (evaldofelipe) [reply .circleci/config.yml:13]: @mrice32 good catch! added to witelist the domains who replies 403 but still working

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**